scripts/commands/updateApiDocs.ts

```diff
@@ -278,9 +316,18 @@ async function convertHtmlToMarkdown(
       html,
       url: `${pkg.baseUrl}/${file}`,
       baseSourceUrl,
-      imageDestination: `/images/api/${pkg.name}`,
+      imageDestination: historical
+        ? `/images/api/${pkg.name}/${versionWithoutPatch}`
+        : `/images/api/${pkg.name}`,
       releaseNotesTitle: `${pkg.title} ${versionWithoutPatch} release notes`,
     });
+
+    result.markdown = transformLinks(result.markdown, (link, _) =>
+      /^\/images.*/.test(link)
```

Comment:
This line should take into account external links and whether we are in the historical version's case or not.